stat/samplemv/halton.go

```diff
@@ -13,15 +13,25 @@ import (
 	"gonum.org/v1/gonum/stat/distmv"
 )
 
-// Haltoner is a wrapper around the Halton sampling generation method.
-type Haltoner struct {
+// Halton is a type for sampling using the Halton sequence from
+// the given distribution. The specific method for scrambling (or lack thereof)
+// is specified by the HaltonKind. If src is not nil, it will be used to generate
+// the randomness needed to scramble the sequence (if necessary). Halton
```

Comment:
I think this needs something saying what happens is src is nil, but that can happen later since this reflects the current wording.